### ISSUE TITLE
JSON Formatter

### ISSUE THREAD

**Issue by chriskapp** [slashtrace/slashtrace#4]
Hi, great package! It would be handy if there was also a formatter which creates a JSON response out of the error message. I could imagine a response like:

```
{
    "success": false,
    "class": "\RuntimeExcetion",
    "message": "",
    "code": 0,
    "trace": [...],
    "request": [...],
    "server": [...]
}
```

There is also [RFC 7807](https://tools.ietf.org/html/rfc7807) which describes such a format. This would simplify to use this handler also for API cases.

**Comment by victorstanciu** [comment]
I really like this! Will definitely look into it soon. Thanks!

**Comment by victorstanciu** [comment]
@chriskapp Hey Christoph, have a look at #6 and let me know what you think before I merge, please :)

**Comment by victorstanciu** [comment]
Merged and published in [1.1.0](https://github.com/slashtrace/slashtrace/releases/tag/1.1.0).

**Comment by chriskapp** [comment]
@victorstanciu, thanks, the PR looks really good, I will try to use the renderer in my project.
